src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  font-size: 12px;
+  line-height: 18px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__product {
+    &-image {
+      display: flex;
+      width: 160px;
+      height: 134px;
+      margin: auto;
+      margin-bottom: 40px;
+    }
+
+    &-name {
+      display: block;
+      font-size: inherit;
+      font-style: normal;
+      font-weight: 500;
+      margin-bottom: 4px;
+    }
+
+    &-code {
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+      color: $color-secondary;
+      margin-bottom: 16px;
+    }
```

Comment:
All of these classes are elements of the card, move it to level up. Fix in all cases